CONTRIBUTING.rst

```diff
@@ -0,0 +1,159 @@
+Contributing to Vulture
+=======================
+
+Creating and cloning a fork
+---------------------------
+
+Fork the repository on GitHub and do the following:
+
+::
+
+    $ git clone https://github.com/jendrikseipp/vulture.git
+    $ cd vulture
+    $ git remote add origin https://github.com/USERNAME/vulture
```

Comment:
I think we should leave an inline comment:
```
git remote add origin https://github.com/USERNAME/vulture  # Replace USERNAME with your Github username
```

Reply:
cc @jendrikseipp

Reply:
I did mention it right below, I'll change that to an inline comment.